packages/opal-common/opal_common/git/tar_file_to_local_git_extractor.py

```diff
@@ -82,10 +84,14 @@ def extract_bundle_tar(self, mode: str = "r:gz") -> bool:
         Args:
             mode(str): mode for TarSafe default to r:gz that can open tar.gz files
         """
-        with TarSafe.open(self.tmp_bundle_path, mode=mode) as tar_file:
-            tar_file_names = tar_file.getnames()
-            TarFileToLocalGitExtractor.validate_tar_or_throw(tar_file_names)
-            tar_file.extractall(path=self.local_clone_path)
+        if opal_common_config.TRUSTED_TAR == True:
```

Comment:
I think we need a way to mark specific TARS as safe explicitly - so people won't by mistake treat unsafe tars as safe.
Kinda like React's `dangerously set innerhtml` 

Maybe we can define a regex for the file name that can act as the identification ( and worst people can config it to ".*" for any file)